spec/features/events_spec.rb

```diff
@@ -118,25 +120,30 @@
     end
   end
 
-  scenario "logged in as Organizer I can change application status with radio buttons" do
+  scenario "logged in as Organizer I can change application status with radio buttons in selection phase" do
     login(:organizer)
-    @event.application_status_locked = false
-    @event.save
     @pupil = FactoryGirl.create(:profile)
     @application_letter = FactoryGirl.create(:application_letter, event: @event, user: @pupil.user)
+    @event.application_status_locked = false
```

Comment:
The problem is that I need a event that is in selection phase and has exactly one application. If a event in selection phase is created no application letter can be added because the deadline is over...